plutus-contract/src/Plutus/Contract/StateMachine.hs

```diff
@@ -422,28 +430,29 @@ runInitialiseWith ::
-runInitialiseWith customLookups customConstraints StateMachineClient{scInstance} initialState initialValue = mapError (review _SMContractError) $ do
-    ownPK <- ownPaymentPubKeyHash
-    utxo <- utxosAt (Ledger.pubKeyHashAddress ownPK Nothing)
-    let StateMachineInstance{stateMachine, typedValidator} = scInstance
-        constraints = mustPayToTheScript initialState (initialValue <> SM.threadTokenValueOrZero scInstance)
-            <> foldMap ttConstraints (smThreadToken stateMachine)
-            <> customConstraints
-        red = Ledger.Redeemer (PlutusTx.toBuiltinData (Scripts.validatorHash typedValidator, Mint))
-        ttConstraints ThreadToken{ttOutRef} =
-            mustMintValueWithRedeemer red (SM.threadTokenValueOrZero scInstance)
-            <> mustSpendPubKeyOutput ttOutRef
-        lookups = Constraints.typedValidatorLookups typedValidator
-            <> foldMap (mintingPolicy . curPolicy . ttOutRef) (smThreadToken stateMachine)
-            <> Constraints.unspentOutputs utxo
-            <> customLookups
-    utx <- mapError (review _ConstraintResolutionContractError) (mkTxContract lookups constraints)
-    adjustedUtx <- adjustUnbalancedTx utx
-    unless (utx == adjustedUtx) $
-      logWarn @Text $ "Plutus.Contract.StateMachine.runInitialise: "
-                    <> "Found a transaction output value with less than the minimum amount of Ada. Adjusting ..."
-    submitTxConfirmed adjustedUtx
-    pure initialState
+runInitialiseWith customLookups customConstraints StateMachineClient{scInstance} initialState initialValue =
+    mapError (review _SMContractError) $ do
+      ownPK <- ownPaymentPubKeyHash
+      utxo <- utxosAt (Ledger.pubKeyHashAddress ownPK Nothing)
+      let StateMachineInstance{stateMachine, typedValidator} = scInstance
+          constraints = mustPayToTheScript initialState (initialValue <> SM.threadTokenValueOrZero scInstance)
+              <> foldMap ttConstraints (smThreadToken stateMachine)
+              <> customConstraints
+          red = Ledger.Redeemer (PlutusTx.toBuiltinData (Scripts.validatorHash typedValidator, Mint))
+          ttConstraints ThreadToken{ttOutRef} =
+              mustMintValueWithRedeemer red (SM.threadTokenValueOrZero scInstance)
+              <> mustSpendPubKeyOutput ttOutRef
+          lookups = Constraints.typedValidatorLookups typedValidator
+              <> foldMap (mintingPolicy . curPolicy . ttOutRef) (smThreadToken stateMachine)
+              <> Constraints.unspentOutputs utxo
+              <> customLookups
+      utx <- mapError (review _ConstraintResolutionContractError) (mkTxContract lookups constraints)
+      adjustedUtx <- adjustUnbalancedTx utx
+      unless (utx == adjustedUtx) $
+        logWarn @Text $ "Plutus.Contract.StateMachine.runInitialise: "
+                      <> "Found a transaction output value with less than the minimum amount of Ada. Adjusting ..."
+      submitTxConfirmed adjustedUtx
+      pure initialState
```

Comment:
There is no implementation change here right? Just formatting?

Reply:
Yes, it should have been a separate commit mentioning formatting. The previous line was ticking at 149 characters which I thought was too much to leave it like that.

```
runInitialiseWith customLookups customConstraints StateMachineClient{scInstance} initialState initialValue = mapError (review _SMContractError) $ do
```

Inserting a newline after the `=` indented the rest.